fe/src/project/Seller.Portal/areas/Products/components/ProductCardForm/ProductCardForm.js

```diff
@@ -189,6 +189,48 @@ const ProductCardForm = (props) => {
         updateSchema(newElements, schema)
     }
 
+    const getIdFromElements = (elements) => {
```

Comment:
**suggestion:** wouldn't it better to generate a random id? https://stackoverflow.com/questions/105034/how-do-i-create-a-guid-uuid

Reply:
Hey, this is so that you can create more new fields, e.g. NewElement1, NewElement2 etc.

I want to use guid for definitions and their values